jax/experimental/jax2tf/README.md

```diff
@@ -9,6 +9,9 @@ TensorFlow eager mode, or stage it out as a TensorFlow graph, even save it
 as a SavedModel for use with TensorFlow tools such as serving stack,
 or TensorFlow Hub.
 
+The package also contains an experimental mechanism to call TensorFlow functions
+from JAX. See [below](#calling-tensorflow-functions-from-jax).
```

Comment:
It does probably make sense to keep the TensorFlow inter-op code all in one place, though this sort of feels like something that should live in a module called `tf2jax` instead of `jax2tf` :)

Reply:
You are right and I went back and forth there. I considered having this a separate experimental module, call_tf. I did not want to call it tf2jax because it does not really turn TF code into first-class JAX code, all you can do is call it and reverse-mode AD. 

For now I thought I keep it part of jax2tf, to make it easier to keep the documentation and tests together. The tests are different from the rest of JAX because they depend on TF, which is not a requirement for the JAX package.